LEGEND.md

```diff
@@ -10,54 +10,39 @@
 
 ![](./assets/img/help_images/web-desktop-keyboard-map.png)
 
-- Note that the chars on the top require shift + key.\*
+- _Note that the chars on the top require shift + key._
 
 ### <a name="slides">Slide Shortcuts</a>
 
-#### MacOS:
+#### macOS:
 
-Cmd + e = empty slide
-Cmd + 1 = Waheguru Slide
+- Cmd + 1 = Waheguru Slide
+- Cmd + 2 = Ik Oankar Slide
+- Cmd + 3 = Empty Slide
+- Cmd + 4 = Anand Sahib (Bhog)
+- Cmd + 5 = Help Guide
+- Cmd + 6 = Shortcut Legend
```

Comment:
Do we really need a shortcut for the help/shortcut guides? I think we can omit these.

Reply:
ok